### ISSUE TITLE
Integration tests fail as there is no device connection service AMQP endpoint

### ISSUE THREAD

**Issue by kaniyan** [eclipse/hono#1975]
Currently I am setting up integration tests to test the _mongo db based device registry_ using a separate _maven_ profile. Since the Infinispan based device connection service is to be used by default, a mongo db based device connection service is not implemented. The integration tests fail with the error (see below) as there is no device connection service AMQP endpoint. But the integration tests with the example _file based device registry_ succeed as it uses the in-memory (`MapBasedDeviceConnectionService`) based implementation of the service.

IMHO how to proceed further with this has also relevance to the currently ongoing discussions on topics device connection and command handling. I raised this ticket so that this issue could also be discussed.
 
```
HTTP17:44:27.407 [vert.x-eventloop-thread-0] INFO  o.e.h.c.i.ProtocolAdapterCommandConsumerFactoryImpl - error setting command handling adapter instance [tenant: a7c3f8ad-c800-4c18-aa9d-4bf4f431b36f, device: f74b1591-da0b-4fa8-a57a-2f66b07a7cd9]
HTTPorg.eclipse.hono.client.ClientErrorException: no endpoint registered for address device_con/a7c3f8ad-c800-4c18-aa9d-4bf4f431b36f/c072427b-06b3-45cd-a6b7-8967e12bb145
```

**Comment by kaniyan** [comment]
This has been addressed in PR #1980.

**Comment by sophokles73** [comment]
@kaniyan can this be closed with #1980 having been merged?

**Comment by kaniyan** [comment]
> @kaniyan can this be closed with #1980 having been merged?

This issue has been closed with #1980.
